Examples/SwiftUIExample/SwiftUIExample/Examples/Buttons/BezierFloatingButton.swift

```diff
@@ -0,0 +1,47 @@
+//
+//  BezierFloatingButton.swift
```

Comment:
```suggestion
//  BezierFloatingButtonExample.swift
```
파일 이름이 컴포넌트와 동일해서 헷갈릴 것 같아요. 파일 내 구조체인 BezierFloatingButtonExample과 동일한 파일명이면 좋지 않을까 싶습니다. BezierFloatingIconButtonExample, BezierIconButtonExample도 동일하게 적용하면 좋겠습니다~